### ISSUE TITLE
Tokihogushi Release

### ISSUE THREAD

**Issue by DoctorDiablo** [07th-mod/higurashi-console-arcs#45]
I just finished translating Tokihogushi, so we can start setting that up.  There's actually not a lot we need to do before release this time.

@enumag Can you run all the toki_ scripts through the ADV compiler.
I'll do an editing pass once that's done.
I also need to package the CG, voice, and BGM archives.  I'll link those here soon.

**Comment by enumag** [comment]
Does this chapter have any new characters? Particularly characters with sprites?

**Comment by enumag** [comment]
We can generate the voice pack btw. I have a script that will copy all the voices actually used in the scripts along with their spectrum files for lipsync.

**Comment by enumag** [comment]
About sprites I believe there was some upgrade recently where @TellowKrinkle compressed all the images. We should make sure to use these compressed sprites instead of whatever we were using before. But I'm not exactly sure if we have a full PS3 pack with these compressed images. cc @ItaloKnox

**Comment by ItaloKnox** [comment]
This is actually a very happy coincidence because yesterday I had this strange urge to remake my compilation folder with the new images. It should be using the new compressed images already. If you want to compile it yourself, I can provide a link to the .7z file with everything already organized. If you leave it to me, I can run it directly in the server.

Either way, if you want just the two separated packs (sprites 1440p and graphics 1080p), the new compressed packs are in the ``higurashi_resources`` folder in our server. I can link everything in the resources channel if you find it more convenient.

**Comment by DoctorDiablo** [comment]
@ItaloKnox @enumag If you guys would like to generate the voice and image packs, I can use them.  I'll diff them with what I have locally to make sure everything is there.  Looks like all the console arcs archives in rikachama are dated Jan 21 or earlier, so we haven't actually released the updated packs @tellowkrinkle put together yet.

@enumag There are a couple new characters in this chapter:
Nagisa, 35, nagisa_
Tsukada, 33, tsukada_
Hanada, 32, hana_

Also, some existing characters got additional sprite variants
Madoka, 29, mado2_
Natsumi, 36, na3_

**Comment by enumag** [comment]
@ItaloKnox Here are the compilation scripts for Tokihogushi, both sprites and voices.

[toki.zip](https://github.com/07th-mod/higurashi-console-arcs/files/3100197/toki.zip)

Beware that it's for Tokihogushi only. Sprites and voices for the previous console arcs are not included. Let me know if you need a full script instead.

**Comment by ItaloKnox** [comment]
@DoctorDiablo You sure we haven't? Because I already synced his new packs over the old ones a while ago. I even gave a deadline on Discord to stop me from doing it in case someone was against it.

The files have the same date as the ones in the developer folder, so I'm pretty sure I didn't miss anything.

@enumag I'll be running those a little later. In any case, I can just merge it with the current archive if that's okay.

**Comment by DoctorDiablo** [comment]
@ItaloKnox Looks like you're right.  I just compared the ConsoleArcs-CG archive with the higurashi_resources pack and they both have the updated files, so we did release them.  I guess I was just confused because the archives had a timestamp from January.  Was it really that long ago?

**Comment by ItaloKnox** [comment]
Yeah, I'm also surprised it stayed that long without being pushed to production. At some point, I was done with waiting for someone to test it "properly" so I just went and pushed it. lol

Here are the packs, beware that those are *partial*, so they aren't merged with the previous chapters yet.
http://172.110.6.39/ddl/Tokihogushi-CG.7z
http://172.110.6.39/ddl/Tokihogushi-Voices.7z

**Comment by DoctorDiablo** [comment]
@ItaloKnox Everything is staged and ready to go.  Three archives were updated.  I pushed them to the developer folder on the server and they need to be pushed to rikachama: ConsoleArcs-BGM, ConsoleArcs-CG, and ConsoleArcs-Voices
I put together a [draft release](https://github.com/07th-mod/higurashi-console-arcs/releases/tag/untagged-d5b8a600d4a29ebb5e84) that you can push once everything is ready.

We should be good to release whenever @drojf says the installer is ready.

**Comment by drojf** [comment]
I think I've resolved the issue that was reported with the Console Arcs installer - the only part which failed was copying the .exe at the very end of the install.

I also tried installing Console Arcs, running it, overwriting it with Ch 4, running it, then overwriting it with Console Arcs again, and running it, and all times the game booted and seemed to run fine when i skipped through a bit.

So I think we're good to go. If there are further issues I'll be on standby to fix them.
